src/server/api/terrain.js

```diff
@@ -106,5 +116,7 @@ export const call = (
         };
     }
 
-    return fetch(apiURL, requestOptions);
+    requestOptions.url = apiURL.toString();
+
+    return axios(requestOptions);
```

Comment:
Will changing this from `fetch` to `axios` break the `fetchMock` calls in the stories?

Reply:
Good catch! Test seems to be broken but I think I found a `axios` mock lib - https://www.npmjs.com/package/axios-mock-adapter. I will update the tests.